metaci/testresults/robot_importer.py

```diff
@@ -294,23 +319,11 @@ def export_robot_test_results(flowtest) -> None:
         "build": {
```

Comment:
We need to include the org name - would that be `flowtask.build_flow.build.org.name`?

Reply:
Yes but then I think you need to change the GUS-BUS model to move it out of the BuildMetadata and into the Build Pydantic model here https://github.com/SFDO-Tooling/gus-bus/blob/main/gus_bus/testmanager.py#L35

Let's address everything else then we can modify this, since it's such a small change. Lets just make sure we don't forget.

Reply:
Good catch. Yes, you're right. Good catch.